client/jetpack-cloud/sections/sidebar-navigation/purchases.tsx

```diff
@@ -1,11 +1,78 @@
+import { chevronLeft, formatListBulletsRTL, payment, receipt, store, tag } from '@wordpress/icons';
+import { useTranslate } from 'i18n-calypso';
+import page from 'page';
 import NewSidebar from 'calypso/jetpack-cloud/components/sidebar';
+import { itemLinkMatches } from 'calypso/my-sites/sidebar/utils';
+
+const PARTNER_PORTAL_ROOT_PATH = '/partner-portal';
```

Comment:
Can we now move this to `client/jetpack-cloud/sections/sidebar-navigation/lib/constants.ts`